### PR TITLE
Add `AGPreferredContentSizeCategoryMocker` tests.

### PR DESCRIPTION
**Pull request by Grubas7**

### CODE REVIEW

File: AGSnapshotHelperTestApp/ViewController.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+//  Created by Adam Grzegorowski on 27/11/15.
+//  Copyright (c) 2015 allegro. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ViewController : UIViewController
+
+
```
Comment (bielski) [inline AGSnapshotHelperTestApp/ViewController.h]: Line not needed.

File: AGSnapshotHelperTestApp/ViewController.m [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view, typically from a nib.
```
Comment (bielski) [inline AGSnapshotHelperTestApp/ViewController.m]: Unnecessary comments.

File: AGSnapshotHelperTests/AGPreferredContentSizeCategoryMockerTests.m [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    // Arrange
+    NSString *contentSizeCategoryBeforeMocking = [UIApplication sharedApplication].preferredContentSizeCategory;
+    NSString *contentSizeCategoryToMock = [self contentSizeCategoryDifferentThanPreferred];
+
+    AGPreferredContentSizeCategoryMocker *mocker = [[AGPreferredContentSizeCategoryMocker alloc] init];
+    [mocker startMockingPreferredContentSizeCategory:contentSizeCategoryToMock];
+
+    // Act
+    [mocker stopMockingPreferredContentSizeCategory];
```
Comment (bielski) [inline AGSnapshotHelperTests/AGPreferredContentSizeCategoryMockerTests.m]: Shouldn't be startMockingPreferredContentSizeCategory?

File: AGSnapshotHelperTests/AGPreferredContentSizeCategoryMockerTests.m [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+@interface AGPreferredContentSizeCategoryMockerTests : XCTestCase
+
+@end
+
+@implementation AGPreferredContentSizeCategoryMockerTests
+
+- (void)testPreferredContentSizeCategoryShouldBeEqualToSetValueAfterStartingMocking {
```
Comment (bielski) [inline AGSnapshotHelperTests/AGPreferredContentSizeCategoryMockerTests.m]: Maybe testPreferredContentSizeCategoryShouldBeEqualToSetValueAfterStarMocking? Up to you.

File: AGSnapshotHelperTestApp/AppDelegate.m [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
```
Comment (bielski) [inline AGSnapshotHelperTestApp/AppDelegate.m]: Unnecessary comments.

**Comment by bielski** [comment]
Review done.

**Comment by Grubas7** [comment]
Fixed, and squashed.

**Comment by bielski** [comment]
:+1:

**Pull request merged**